README.md

```diff
@@ -346,6 +351,35 @@ command:
 xcodebuild build CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -project "Mozilla VPN.xcodeproj"
 ```
 
+#### Building with CMake (Experimental)
+
+We also support building from sources for iOS using CMake. This is a work in progress and these
+instructions may be subject to change as we figure out what works and what doesn't.
```

Comment:
```suggestion
We also support building from sources for iOS using CMake.
```

Isn't the WIP and "please update if things change" true of all our documentation?